srml/democracy/src/vote_threshold.rs

```diff
@@ -42,7 +44,9 @@ pub trait Approved<Balance> {
 }
 
 /// Return `true` iff `n1 / d1 < n2 / d2`. `d1` and `d2` may not be zero.
-fn compare_rationals<T: Zero + Mul<T, Output = T> + Div<T, Output = T> + Rem<T, Output = T> + Ord + Copy>(mut n1: T, mut d1: T, mut n2: T, mut d2: T) -> bool {
+fn compare_rationals<T>(mut n1: T, mut d1: T, mut n2: T, mut d2: T) -> bool
+	where T: Zero + Mul<T, Output = T> + Div<T, Output = T> + Rem<T, Output = T> + Ord + Copy
```

Comment:
suggest this be included in a separate PR since it changes the implementation (instead of mixed into a documentation-specific PR)

Reply:
It does not change anything really, it is just a reformat of the code